modin/engines/ray/pandas_on_ray/frame/axis_partition.py

```diff
@@ -13,11 +13,15 @@
 
 import pandas
 
+from modin.config import EnablePartitionIPs
 from modin.engines.base.frame.axis_partition import PandasFrameAxisPartition
 from .partition import PandasOnRayFramePartition
 
 import ray
-from ray.services import get_node_ip_address
+
+
+if EnablePartitionIPs.get():
+    from ray.services import get_node_ip_address
```

Comment:
```suggestion
from ray.services import get_node_ip_address
```
This is not entirely corrent, the config variable could be changed in runtime.
I suggest removing the check here.